element_array_ephys/ephys_acute.py

```diff
@@ -662,6 +664,74 @@ def yield_unit_waveforms():
             self.Waveform.insert(unit_electrode_waveforms, ignore_extra_fields=True)
 
 
+@schema
+class QualityMetric(dj.Imported):
```

Comment:
In this case, the plural form `QualityMetrics` is justified since each row represents multiple metrics.